clippy_lints/src/booleans.rs

```diff
@@ -203,7 +203,7 @@ impl<'a, 'tcx, 'v> SuggestContext<'a, 'tcx, 'v> {
                 METHODS_WITH_NEGATION
                     .iter().cloned()
                     .flat_map(|(a, b)| vec![(a, b), (b, a)])
-                    .find(|&(a, _)| a == path.name.as_str())
+                    .find(|&(a, _)| a == path.ident.name.as_str())
```

Comment:
`ident.as_str()` works too

Reply:
Ah, I made note to myself to check that and forgot about it.